docs/advanced-entity-analytics/asset-criticality.asciidoc

```diff
@@ -26,7 +26,9 @@ For example, you can assign **Extreme impact** to business-critical entities, or
 [discrete]
 == View and assign asset criticality
 
-Entities do not have a default asset criticality level. You can view, assign, change, or unassign asset criticality from the following places in the {elastic-sec} app:
+Entities do not have a default asset criticality level. You can either assign asset criticality to your entities individually, or <<bulk-assign-asset-criticality, bulk assign>> it to multiple entities by importing a text file.
```

Comment:
I feel like the original is a bit clearer ("bulk assign multiple entities" could be interpreted as though users are assigning entities to something rather than assigning asset criticality to entities).